requirements.txt

```diff
@@ -1,3 +1,4 @@
 arrow!=0.11,!=0.12.0
 click
 requests
+click-didyoumean
```

Comment:
I would put both click dependencies togeter.

Reply:
I'm neutral to the style here and let's see if maintainers have any preference.

Reply:
I like when things are alphabetically sorted :wink: